sdk/servicebus/perf-tests.yml

```diff
@@ -0,0 +1,44 @@
+Service: servicebus
+
+Project: sdk/servicebus/azure-servicebus
+
+PrimaryPackage: azure-eventhub
+
+PackageVersions:
+- azure-core: 1.26.1
+  azure-eventhub: 7.8.3
+- azure-core: source
+  azure-eventhub: source
+
+Tests:
+- Test: send-queue-message-batch
+  Class: SendQueueMessageBatchTest
+  Arguments:
+  - --event-size 1024 --batch-size 100 --parallel 64
+  - --event-size 1024 --batch-size 100 --parallel 64 --uamqp-transport
+  - --event-size 1024 --batch-size 100 --parallel 64 --transport-type 1
+  - --event-size 1024 --batch-size 100 --parallel 64 --transport-type 1 --uamqp-transport
+
+- Test: receive-queue-message-batch
+  Class: ReceiveQueueMessageBatchTest
+  Arguments:
```

Comment:
Optional, but you can use yaml anchors/aliases for duplicate argument lists. Example usage: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-blob/perf-tests.yml#L16